e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Test Tutorial Navigation', () => {
+    before(() => {
+        cy.loginAsNewUserWithTutorial();
+    }); // before
```

Comment:
Omit these trailing comments.

Follow the pattern of something like:

```
it('this is my test', () => {
    // # Login as the user
    cy.login('myuser')

    // # Then do more stuff
    cy.doMoreStuff()

    // * Verify something
    cy.verifyThatThing()
})
```

Note the comment block at the top of this file for the guidelines.

Reply:
I like to comment after brackets so I can remember what they're supposed to be closing, but I've followed your direction and have added more standardized commenting.